network_wrangler/scenario.py

```diff
@@ -38,627 +41,530 @@ class Scenario(object):
-            msg = "Cannot find specified directory to add project cards: {}".format(
-                folder
-            )
-            WranglerLogger.error(msg)
-            raise ValueError(msg)
+        Resets scenario queued_projects.
 
-        if glob_search:
-            WranglerLogger.info(
-                "Adding project cards using glob search: {}".format(glob_search)
-            )
-            for file in glob.iglob(os.path.join(folder, glob_search)):
-                if not (
-                    file.endswith(".yml")
-                    or file.endswith(".yaml")
-                    or file.endswith(".wrangler")
-                    or file.endswith(".wr")
-                ):
-                    continue
-                else:
-                    self.add_project_card_from_file(file, validate=validate)
-        else:
-            for file in os.listdir(folder):
-                if not (
-                    file.endswith(".yml")
-                    or file.endswith(".yaml")
-                    or file.endswith(".wrangler")
-                    or file.endswith(".wr")
-                ):
-                    continue
-                else:
-                    self.add_project_card_from_file(
-                        os.path.join(folder, file), validate=validate
-                    )
+        Args:
+            project_card (ProjectCard): ProjectCard instance to add to scenario.
+            validate (bool, optional): If True, will validate the projectcard before
+                being adding it to the scenario. Defaults to True.
+            filter_tags (Collection[str], optional): If used, will only add the project card if
+                its tags match one or more of these filter_tags. Defaults to []
+                which means no tag-filtering will occur.
 
-    def add_project_cards_from_tags(
-        self, folder: str, tags: List[str] = [], glob_search="", validate=True
-    ):
         """
-        Adds projects cards to the scenario.
-        A folder is provided to look for project cards that have a matching tag that
-        is passed to the method.
+        project_name = project_card.project.lower()
+        filter_tags = map(str.lower, filter_tags)
 
-        args:
-        folder: the folder location where the project cards will be
-        tags: only project cards with these tags will be validated and added to the
-            returning scenario
-        """
+        if project_name in self.projects:
+            raise ValueError(
+                f"Names not unique from existing scenario projects: {project_card.project}"
+            )
 
-        if glob_search:
+        if filter_tags and project_card.tags.isdisjoint(filter_tags):
             WranglerLogger.debug(
-                "Adding project cards using \n-tags: {} and \nglob search: {}".format(
-                    tags, glob_search
-                )
+                f"Skipping {project_name} - no overlapping tags with {filter_tags}."
```

Comment:
`filter_tags` is a map object, suggest changing to dictionary.

Reply:
The map object should just be changing the list...but I forgot to change it back to a list so added that and should be good now.

Reply:
it should actually be a list - forgot to wrap the map in a list call...fixed!